tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
```

Comment:
It seems `esbuild` does not support the new JSX transform yet so I've reverted this.

Reply:
What was the issue?

Reply:
It seems that esbuild has to add support for the React 17 JSX transform so I guess it just wasn't going to build with `"react-jsx"` (there were conflicting errors about needing React to be imported/not imported).

More details in the issue: https://github.com/evanw/esbuild/issues/334.